ui/app/models/identity/group.js

```diff
@@ -36,19 +36,28 @@ export default IdentityModel.extend({
     editType: 'kv',
   }),
   policies: attr({
-    editType: 'stringArray',
+    label: 'Policies',
+    editType: 'searchSelect',
+    fallbackComponent: 'string-list',
+    models: ['policy/acl', 'policy/rgp'],
   }),
   memberGroupIds: attr({
     label: 'Member Group IDs',
-    editType: 'stringArray',
+    editType: 'searchSelect',
+    fallbackComponent: 'stringArray',
+    models: ['identity/group'],
   }),
   parentGroupIds: attr({
     label: 'Parent Group IDs',
-    editType: 'stringArray',
+    editType: 'searchSelect',
+    fallbackComponent: 'string-list',
```

Comment:
not something we need to do here, but thinking if we get more of these complex types, we may want to standardize the config here - maybe `editType` is an object that can take use the keys for config? What do you think?